pkg/services/star/componentroot.yaml

```diff
@@ -0,0 +1,4 @@
+groups:
+  - domain
+owners:
+  - "@grafana/backend-platform"
```

Comment:
what is owners doing? that means we are going to be reviews when some rules are applied? what are those rules

Reply:
ah yeah, sorry, forgot to explain that bit. 

The intention there is to also enforce that all components have owners, then be able to compile all that down into a CODEOWNERS file. GitHub requires those to all be in a single file at/near the root of the repo, so this was a way to have just one more thing we can encapsulate within the component.

(see `compgen codeowners gen` iirc)

Right now the tool is hardcoded to work with CODEOWNERS at the root, though, not in .github like ours is, so it needs a little tweaking.